R/pmcmc_control.R

```diff
@@ -99,6 +99,12 @@
 ##' @param progress Logical, indicating if a progress bar should be
 ##'   displayed, using [`progress::progress_bar`].
 ##'
+##' @param nested_step_ratio Either integer or 1/integer, which specifies the
+##'   ratio of fixed:varied steps in a nested pMCMC. For example `3` would run
+##'   3 steps proposing fixed parameters only and then 1 step proposing varied
+##'   parameters only; whereas `1/3` would run 3 varied steps
+##'   for every 1 fixed step.
```

Comment:
Add something like:

> The default value of 1 runs an equal number of iterations updating the fixed and varied parameters.

You might also want to provide guidance as to how to select a sensible value here based on the number of fixed and varied parameters